subsys/caf/modules/sensor_sampler.c

```diff
@@ -294,38 +310,39 @@ static int sensor_init(void)
 		if (sc->trigger) {
 			err = sensor_trigger_init(sc, sd);
 			if (err) {
```

Comment:
Should we still break on error here? Sensors were meant to be independent.
We could return number of initialized sensors or negative value on error here.

Reply:
I think we should. This means there is an configuration error user should fix.